array.go

```diff
@@ -75,6 +75,15 @@ func (a *Array) Path(path string) *Value {
 	return jsonPath(a.chain, a.value, path)
 }
 
+// Alias is similar to Value.Alias.
+func (a *Array) Alias(name string) *Array {
```

Comment:
Here and in other structs: let's place Alias before assertion methods, right after Raw.

Reply:
I fixed on 534ab72 .